src/views/lighttable.c

```diff
@@ -3137,7 +3156,7 @@ static void _culling_scroll(dt_library_t *lib, const int up)
     {
       if(lib->culling_use_selection)
         dt_control_log(_("you have reached the start of your selection (%d images)"),
```

Comment:
I suppose here (and below) we want a string for handling single image or multiple (singular / plural).
see ngettext() usage.

Reply:
done